src/main/java/com/google/api/codegen/transformer/go/GoSurfaceNamer.java

```diff
@@ -234,7 +234,11 @@ Name clientNamePrefix(Interface service) {
 
   @Override
   public String getStatusCodeName(Status.Code code) {
-    return publicFieldName(Name.upperUnderscore(code.toString()));
+    String codeString = code.toString();
+    if (code.equals(Status.Code.CANCELLED)) {
+      codeString = "CANCELED";
```

Comment:
There's currently no test for this, but it seems silly to add a CANCELLED to some method, since it will change baselines for all languages. @garrettjonesgoogle WDYT?